package.json

```diff
@@ -88,7 +88,7 @@
     "webpack-cli": "^4.5.0"
   },
   "engines": {
-    "node": ">=10.18.0"
+    "node": ">=12.20.0"
```

Comment:
NIT: I would go with `"node": "^12.20.0 || ^14.13.1 || >=16.0.0"` for esm support missed that on the CLI.

Regarding to sindresorhus: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c